server/utils/db.ts

```diff
@@ -0,0 +1,37 @@
+import { UserSchema } from '../models/user.model.ts';
+import { RoomSchema } from '../models/room.model.ts';
+import { MongoClient, ObjectId, config } from '../deps.ts';
+config({ path: './.env' });
+
+const connectToDb = async () => {
```

Comment:
According to MongoDB best practices, you're supposed to open the connection once when the app starts, which creates a connection (actually a pool of connections for increased efficiency), and then close it only when the application is shutting down (at least for Node - I couldn't find any Deno-specific resources for this).

Therefore, I'm thinking we should just open it once on startup, rather than making a util to create a connection and closing it each time.

Or am I misunderstanding?